nautobot/core/views/mixins.py

```diff
@@ -909,7 +912,41 @@ def perform_update(self, request, *args, **kwargs):  # pylint: disable=arguments
             return self.form_invalid(form)
 
 
-class ObjectBulkDestroyViewMixin(NautobotViewSetMixin, BulkDestroyModelMixin):
+class DeleteAllModelMixin:
+    """
+    UI mixin to bulk destroy all model instances.
+    """
+
+    def _get_bulk_delete_all_queryset(self, request):
```

Comment:
Docstrings on these functions please - yes they're "hidden" but they're an important part of how these views work.